src/Provider/FileProvider.php

```diff
@@ -315,9 +315,9 @@ public function validate(ErrorElement $errorElement, MediaInterface $media)
 
         if ($media->getBinaryContent() instanceof UploadedFile && 0 === ($media->getBinaryContent()->getClientSize() ?: 0)) {
             $errorElement
-               ->with('binaryContent')
-                   ->addViolation('The file is too big, max size: '.ini_get('upload_max_filesize'))
-               ->end();
+                ->with('binaryContent')
```

Comment:
Please revert thise

Reply:
I pushed a commit that revert this indent.